app/vacancy_sources/vacancy_source/source/every.rb

```diff
@@ -42,7 +42,7 @@ def attributes_for(item)
       salary: item["salary"],
       expires_at: Time.zone.parse(item["expiresAt"]),
       external_advert_url: item["advertUrl"],
-      job_role: job_role(item),
+      job_roles: job_roles(item),
```

Comment:
This ATS source is still not running in production. Added the mapping to correspond with [the PR introducing Every integration](https://github.com/DFE-Digital/teaching-vacancies/pull/5799). Will adapt the tests in that PR (or here if 5799 gets merged first).